### DISCUSSION TITLE
Slider component in next 13

### DISCUSSION THREAD

**Discussion by Friduwulf** [saadeghi/daisyui#1845] [Q&A]
Hello everyone! I'm somewhat new to Next 13 and I am currently trying to use a DaisyUI slider component. When I attempt to use the slider in my application, I cannot move it. I'm sure it's a simple fix, so if anyone can tell me what I'm missing, that would be great. Thanks!

**Comment by saadeghi** [comment]
It's not easy to help when you don't provide any reproduction code, my friend.  
- Maybe the input is disabled (either by a `disabled` attribute or by JS)
- Maybe you have some CSS to prevents interaction (like `pointer-events: none`) or another element is on top of the input.
- Maybe It's actually a JS issue that forces the input value to something else.

Can be anything. Please provide a code example on StackBlitz or CodeSandbox, etc.
